src/Components/HomePage.js

```diff
@@ -2,53 +2,129 @@ import React from 'react';
 import { StyleSheet, View, Text, ImageBackground, TouchableHighlight } from 'react-native';
 
 import theme from '../styles/theme.style';
-import fitness from '../../assets/HomePage/fitness.png';
-import frisbeeGlove from '../../assets/HomePage/frisbeeglove.png';
+import frisbeeGlove from '../../assets/HomePage/frisbeeglove.jpg';
+import bodyweight from '../../assets/HomePage/bodyweight.jpg';
+import gymstrong from '../../assets/HomePage/gymstrong.jpg';
+import leanfit from '../../assets/HomePage/leanfit.jpg';
+import simulator from '../../assets/HomePage/simulator.jpg';
 import huddle from '../../assets/HomePage/huddle.png';
 import { DrillTypes } from '../Fixtures';
-import Button from './shared/Button';
 
-export default class HomePage extends React.Component {
-  render() {
-    return (
-      <View style={styles.mainContainer}>
-        <TouchableHighlight
-          onPress={() => this.props.navigation.navigate('DrillListPage', { type: DrillTypes.FRISBEE })}
-          style={styles.menuItem}
-        >
-          <ImageBackground source={frisbeeGlove} style={styles.image}>
-            <View style={styles.wrapper}>
-              <Text style={styles.text}>Frisbee drills</Text>
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import { Ionicons } from '@expo/vector-icons';
+
+const Tab = createMaterialBottomTabNavigator();
+
+const HomeScreen = props => {
+  return (
+    <View style={styles.mainContainer}>
+      <TouchableHighlight onPress={() => props.navigation.navigate('AnimationEditorPage')} style={styles.menuItem}>
+        <ImageBackground source={simulator} style={styles.image}>
+          <View style={styles.wrapper}>
+            <Text style={styles.text}>Theory</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+      <TouchableHighlight
+        onPress={() => props.navigation.navigate('DrillListPage', { type: DrillTypes.FRISBEE })}
+        style={styles.menuItem}
+      >
+        <ImageBackground source={frisbeeGlove} style={styles.image}>
+          <View style={styles.wrapper}>
+            <Text style={styles.text}>Drills</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+      <TouchableHighlight onPress={() => props.navigation.navigate('ProgramListPage')} style={styles.menuItem}>
+        <ImageBackground source={huddle} style={styles.image}>
+          <View style={styles.wrapper}>
+            <Text style={styles.text}>Practices programs</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+const Fitness = props => {
+  return (
+    <View style={styles.mainContainer}>
+      <TouchableHighlight
+        onPress={() => props.navigation.navigate('DrillListPage', { type: DrillTypes.FITNESS })}
```

Comment:
what are these going to be? Direct links to fitness programs? Or pre-filtered fitness drills list?

Reply:
What I have in mind is direct link to fitness program (that we have to create)